packages/core/scripts/setup/uninstall.ts

```diff
@@ -0,0 +1,337 @@
+import EmbraceLogger from '../../src/logger';
+const fs = require('fs');
+
+import {
+  embraceJSON,
+  getBuildGradlePatchable,
+  getMainApplicationPatchable,
+} from '../util/android';
+import { FileUpdatable } from '../util/file';
+import {
+  bundlePhaseRE,
+  EMBRACE_IMPORT_OBJECTIVEC,
+  EMBRACE_INIT_OBJECTIVEC,
+  embraceNativePod,
+  embracePlistPatchable,
+  exportSourcemapRNVariable,
+  getAppDelegateByIOSLanguage,
+  getPodFile,
+  xcodePatchable,
+} from '../util/ios';
+import Wizard, { Step } from '../util/wizard';
+import { androidEmbraceSwazzlerPlugin, androidGenericVersion } from './android';
+import { findNameWithCaseSensitiveFromPath } from './ios';
+import { MAIN_CLASS_BY_LANGUAGE } from './patches/common';
+
+const packageJson = require('../../../../../../package.json');
+
+export const EMBRACE_IMPORT_SWIFT = 'import Embrace';
+
+export const EMBRACE_INIT_SWIFT =
+  'Embrace.sharedInstance().start(launchOptions: launchOptions, framework:.reactNative)';
+
+export const EMBRACE_IMPORT_JAVA =
+  'import io.embrace.android.embracesdk.Embrace;';
+
+export const EMBRACE_INIT_JAVA =
+  'Embrace.getInstance().start(this, false, Embrace.AppFramework.REACT_NATIVE);';
+
+export const EMBRACE_IMPORT_KOTLIN =
+  'import io.embrace.android.embracesdk.Embrace';
+
+export const EMBRACE_INIT_KOTLIN =
+  'Embrace.getInstance().start(this, false, Embrace.AppFramework.REACT_NATIVE)';
+
+const logger = new EmbraceLogger(console);
+
+interface ITextToDelete {
+  ITextToDelete: string | RegExp;
+}
+
+type FindFileFunction = () => FileUpdatable | undefined;
+
+interface IUnlinkEmbraceCode {
+  stepName: string;
+  fileName: string;
+  textsToDelete: ITextToDelete[];
+  findFileFunction: FindFileFunction;
+  docUrl: string;
+}
+
+const UNINSTALL_IOS_SWIFT_APPDELEGATE: IUnlinkEmbraceCode = {
```

Comment:
these should share a common definition with what is in packages/core/scripts/setup/patches/patch.ts rather than being defined again since whatever text we're adding there should be removed here